calico_containers/tests/st/docker_host.py

```diff
@@ -15,16 +15,21 @@ def __init__(self, name):
         pwd = sh.pwd().stdout.rstrip()
         docker.run("--privileged", "-v", pwd+":/code", "--name", self.name, "-tid", "jpetazzo/dind")
```

Comment:
Yep, I'm very much with @Symmetric here. It _is_ a common pattern to mount code into the container. Docker themselves do it for their build containers.